src/NHibernate.Test/TestsContext.cs

```diff
@@ -58,6 +49,35 @@ private static void ConfigureLog4Net()
 			hierarchy.Root.AddAppender(consoleAppender);
 			hierarchy.Configured = true;
 		}
+
+		class NetCoreConfigurationManager : IConfigurationManager
+		{
+			private readonly System.Configuration.Configuration _configuration;
+
+			public NetCoreConfigurationManager()
+			{
+				var assemblyPath =
```

Comment:
I think this can become a parameter. And then the configuration manager moved to the main code base.

Reply:
Done